### ISSUE TITLE
Add common units

### ISSUE THREAD

**Issue by edvinf** [StoXProject/RstoxData#175]
Add common units vocabulary on load (initialize), using the units library. Consider what happens with updates to the units library (use remove_units with warning if units already exists).

**Comment by edvinf** [comment]
We have discussed using the units package, but may need to add some units and perhaps change the definition of some. The idea is that we can leverage conversion-functionality and the general framework of units, while adapting units conventions to fisheries science. We discussed defining common units in RstoxData and doing necessary modifications in .onLoad so that they would automatically be available in other stox packages that load RstoxData. I have experimented a bit with this approach and have some concerns that I would like to discuss before proceeding.

Overriding the units definition may cause problems when other packages or scripts loads RstoxData. Subsequent calls to units:: will use the RstoxData definitions. It is quite likely that RstoxData get introduced to an existing R-script for some parsing purposes or similar, and redefining any units can break such a script. Likewise, R-objects that are saved and will look like they can be used with the standard unit-configuration when loaded on a different computer, but this may not be the case.

We can try to avoid redefinitions, but that is rather restrictive, and it does not guarantee against any changes that may occur for if we install updates to the units package. It is also quite tricky to ensure that we avoid redefinition, because of the flexible notation that allows prefixing units with common multipliers like 'k' for kilo. For instance, removing the alias 'kt' for knots (aviation convention) "activates" the prefix 'k' for the already existing unit 'metric_tonne', with symbol 't'. This single deletion will completely change subsequent calls to units::set_unit(x,"kt") from meaning knots (speed) to kilotonnes (mass). Similarly adding an alias starting with a letter that is also a prefix could "deactive" a previous valid prefix-notation.

For these reasons I think we have to make sure that we control exactly the unit-table available. At the same time that is exactly what may cause problems if RstoxData is mixed with other scripts or packages.

I will suggest that we look for simpler solutions. We have already agreed that we want our data formats to have fixed units, and that unit conversion is a matter for reporting-functions. What we mainly need is a common vocabulary of units and symbols that ensures consitent presentation to the user, and a way for units to follow the reports, so that units can be reported in output files and any functions that takes a report as an input can know what units fields are in. Secondary to that I think it would be nice with some general conversion methods, but I don't think it is worth sacrificing the independence between packages.
